### PR TITLE
Image and input

### PR DESCRIPTION
**Pull request by Jia-A**
Screenshot (421)
Screenshot (422)
Screenshot (423)
Screenshot (424)
This is my input and image component. I have yet to change px to rems and add the carbonn code.

### CODE REVIEW

File: components/image-and-input/image.css [hunk 1]
```diff
@@ -0,0 +1,47 @@
+@import "../../assets/common-css/sidebar.css";
+@import "../../assets/common-css/navbar.css";
+@import "../../assets/common-css/containers.css";
+@import "../../document-css/doc.css";
+
+.image-head, .image-type{
```
Comment (junaidshaikh-js) [inline components/image-and-input/image.css]: each selector should be on new line

File: components/image-and-input/image.html [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Montserrat:ital,wght@0,300;0,400;0,500;0,900;1,300&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="./image.css">
+    <title>Tec{h}tonic</title>
```
Comment (junaidshaikh-js) [inline components/image-and-input/image.html]: You can modify the title name here like - **Tec{h}tonic | Image**

File: components/image-and-input/image.css [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+}
+.image-head{
+    margin: -5px;
+}
+.image-type{
+    margin-top: 30px;
+}
+.image-content, .res-img-content{
+    padding: 20px;
+    font-size: 19px;
```
Comment (lannisterrr) [inline components/image-and-input/image.css]: you should use rem for font-sizes.

File: components/image-and-input/input.css [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+.invalid-message{
+    color: var(--warning-color);
+    font-size: small;
+    margin-top: 5px;
+}
+.search-box{
+    border-radius: 50px;
+    border: 1px solid var(--primary-color);
+}
+/* .search-bar-input{
```
Comment (junaidshaikh-js) [inline components/image-and-input/input.css:57]: code comments should be removed

File: components/image-and-input/input.html [hunk 5]
```diff
@@ -0,0 +1,105 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Tec{h}tonic</title>
+</head>
+<body>
+    <!DOCTYPE html>
```
Comment (junaidshaikh-js) [inline components/image-and-input/input.html]: Looks like you copied the code twice

**Review by junaidshaikh-js — COMMENTED** [review]
Make use of semantic HTML elements in your code, `header` `main` and `footer` for overall structure.

**Pull request merged**